### DISCUSSION TITLE
CityJson (CityGML) objects visualization

### DISCUSSION THREAD

**Discussion by bbl** [visgl/deck.gl#6403] [Q&A]
Hello! Is it possible to visualize CityJson objects with deck.gl? If not, is it still possible to draw different roof types on the buildings (not just flat ones)?

Example:

![image](https://user-images.githubusercontent.com/15376942/141969875-86f5cb97-b362-4a52-b579-0353fbdc60f4.png)

**Comment by ashvanierssel** [comment]
Bump!
